### PR TITLE
Rickroll post

### PR DESCRIPTION
**Pull request by rsaunders100**

### CODE REVIEW

File: _posts/2017-05-12-how-to-un-rickroll-yourself.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+I had being playing around with a feature to share a restaurant menu on iOS over bluetooth.  I had found it happens without a prompt sometimes.  I'm not sure exactly how they have done it, but I am quite tired so, I just disable bluetooth and go home.
+
+The next day it happens again.  I am much more alert and up for a challenge.
+
+First, I check my chrome extensions. Everything looks normal, but I check the author of each extension to make sure that the rickroll is not [masquerading as google docs](http://www.popsci.com/massive-phishing-scheme-just-hijacked-gmail-accounts-by-disguising-itself-as-shared-google-doc).
+
+I check to see if there are any [cron jobs](https://www.cyberciti.biz/faq/linux-show-what-cron-jobs-are-setup/) set up on my machine - nothing.  I check my downloads, installed applications and terminal history (`history 100`).  Nothing sticks out.
+
+I figure somewhere sitting in a file is the rickroll youtube URL [https://www.youtube.com/watch?v=dQw4w9WgXcQ](https://youtu.be/2Z4m4lnjxkY?t=1m46s) so I start to grep everything for the youtube ID
```
Comment (jphastings) [inline _posts/2017-05-12-how-to-un-rickroll-yourself.md]: <3

Comment (rsaunders100) [inline _posts/2017-05-12-how-to-un-rickroll-yourself.md]: Haha - the new rick roll:

Have a look at this rickroll it's really interesting...

**Comment by rsaunders100** [comment]
@jphastings - I think its ready to go now

**Comment by rsaunders100** [comment]
The date is wrong now.  Not sure if its a problems

**Comment by jphastings** [comment]
This is great, I'll post now :)

**Pull request merged**